styles.css

```diff
@@ -1,125 +1,130 @@
-html{
-  background-color: #faf5e4;
+:root {
+  --gray-color: hsl(0, 0%, 50%);
+  --white-color: hsl(0, 0%, 100%);
+  --very-dark-blue-color: hsl(212, 21%, 14%);
```

Comment:
w tym przypadku takie stopniowanie kolorów jest jak najbardziej w porządku, w większych apkach będziesz korzystać z design systemów, w których stopniuje się kolory za pomocą wartości 50, 100...900, jak np tutaj:
https://mui.com/material-ui/customization/color/#color-palette
Wrzucam jako zajawkę, to będziemy robić przy Reacie :D